scripts/fix_cc_deps.py

```diff
@@ -94,9 +95,11 @@ def remap_file(label: str) -> str:
     repo, _, path = label.partition("//")
     if not repo:
         return path.replace(":", "/")
-    assert repo in EXTERNAL_REPOS, repo
-    return EXTERNAL_REPOS[repo].remap(path)
-    exit(f"Don't know how to remap label '{label}'")
```

Comment:
Right now, it looks like this exit is unreachable. The assert is just there as a safety check to expose the `repo` value.

Can you clarify what issue you ran into that you think it's better to expose the label versus just removing the unreachable exit?

Reply:
I haven't encountered any issues so far, but I just wanted to highlight that the exit statement in the code is currently not reachable.

Reply:
Okay, I think it'd make sense to just remove the unreachable code.

Reply:
Got it, i just removed the unreachable code. :)